cli/src/main.rs

```diff
@@ -4,98 +4,90 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
-use clap::{App, Arg, SubCommand};
-
 use runtime::config::{Config, CONFIG, LogLevel};
-
+use structopt::StructOpt;
```

Comment:
The imports here should be reformatted.
The order in the repository currently is:
1. External Crates
2. Local Crates
3. Within Crate

```rs
// License text here

use structopt::StructOpt;

use runtime::config::{Config, CONFIG, LogLevel};

use crate::commands::{repl, run};
// More imports
```

Reply:
https://github.com/rust-lang/rustfmt/issues/2979

Reply:
I actually use CLion for that, which is why my `rustfmt.toml` says not to touch imports.